frontend/public/components/notification-drawer.tsx

```diff
@@ -108,20 +111,37 @@ const getUpdateNotificationEntries = (
   isLoaded: boolean,
   updateData: ClusterUpdate[],
   isEditable: boolean,
+  newerChannel: string,
+  newerChannelVersion: string,
   toggleNotificationDrawer: () => void,
 ): React.ReactNode[] =>
   isLoaded && !_.isEmpty(updateData) && isEditable
     ? [
         <NotificationEntry
           actionPath="/settings/cluster"
           actionText="Update cluster"
-          key="cluster-udpate"
+          key="cluster-update"
           description={updateData[0].version || 'Unknown'}
           type={NotificationTypes.update}
           title="Cluster update available"
           toggleNotificationDrawer={toggleNotificationDrawer}
           targetPath="/settings/cluster"
         />,
+        newerChannel && (
+          <NotificationEntry
+            actionPath="/settings/cluster?showChannels"
+            actionText="Edit channel"
+            key="channel-update"
+            description={`OpenShift ${newerChannelVersion} is available. If you are
+            interested in updating this cluster to ${newerChannelVersion} in the
+            future, change the update channel to ${newerChannel} to receive recommended
+            updates.`}
+            type={NotificationTypes.update}
+            title={`OpenShift ${newerChannelVersion} is available`}
+            toggleNotificationDrawer={toggleNotificationDrawer}
+            targetPath="/settings/cluster?showChannels"
```

Comment:
The other update notification takes you to the Cluster Settings page, but doesn't open the dialog. We probably want to be consistent. cc @megan-hall

Reply:
I was thinking about this too. 

Do we want users to be able to view the details for the update before navigating them to the update modal? Or do you think it is ok to open the update dialog directly from the notification?

Reply:
Another consideration is the fact we link to `/settings/cluster` several other places.  Do we want consistency when linking?  

I agree with @spadgett that these two notifications should have a consistent behavior, however.

Reply:
I've made both notifications open the appropriate modal, but am happy to change.

Reply:
Agree as well that the two notifications should be consistent. Looks like we link directly to the update modal on the cluster overview today. So I am good with opening dialogs directly for those two alerts. Makes sense.

And totally agree they should be consistent - should have articulated that better :)

Reply:
I think in certain places its ok to link to `/settings/cluster `like on the cluster overview details card in the top right hand corner where it says View details. But the about modal alert should probably match the notification drawer alert and open the dialog directly. Is that what you were referring to @rhamilto?

Reply:
Exactly, @megan-hall.  We're in agreement.